changelog.d/5465.bugfix

```diff
@@ -0,0 +1 @@
+Fix a crash happening when running a specific background update.
```

Comment:
Actually, can you just use the same changelog text (and type) as the original PR so their entries get merged?

Reply:
Sure